
[I 2023-06-19 05:40:14,463] A new study created in memory with name: no-name-9295d9f7-b24f-4157-bb8c-f7c8a596049f
c:\Users\caiay\AppData\Local\Programs\Python\Python310\lib\site-packages\torchvision\models\_utils.py:208: UserWarning: The parameter 'pretrained' is deprecated since 0.13 and may be removed in the future, please use 'weights' instead.
  warnings.warn(
c:\Users\caiay\AppData\Local\Programs\Python\Python310\lib\site-packages\torchvision\models\_utils.py:223: UserWarning: Arguments other than a weight enum or `None` for 'weights' are deprecated since 0.13 and may be removed in the future. The current behavior is equivalent to passing `weights=ResNet18_Weights.IMAGENET1K_V1`. You can also use `weights=ResNet18_Weights.DEFAULT` to get the most up-to-date weights.
  warnings.warn(msg)
[I 2023-06-19 06:15:06,986] Trial 0 finished with value: 0.5110833048820496 and parameters: {'learning_rate': 0.00010758111015124252, 'dropout_rate': 0.3579861844634901}. Best is trial 0 with value: 0.5110833048820496.
[I 2023-06-19 06:40:04,589] Trial 1 finished with value: 0.722083330154419 and parameters: {'learning_rate': 0.0009673168340862709, 'dropout_rate': 0.49397238347718947}. Best is trial 1 with value: 0.722083330154419.
[I 2023-06-19 07:03:21,049] Trial 2 finished with value: 0.8339999914169312 and parameters: {'learning_rate': 0.020692901318195428, 'dropout_rate': 0.04479207377464206}. Best is trial 2 with value: 0.8339999914169312.
[I 2023-06-19 07:25:00,959] Trial 3 finished with value: 0.5673333406448364 and parameters: {'learning_rate': 0.0002191185367478793, 'dropout_rate': 0.28188130592342114}. Best is trial 2 with value: 0.8339999914169312.
[I 2023-06-19 07:46:11,166] Trial 4 finished with value: 0.8355000019073486 and parameters: {'learning_rate': 0.030885685338495866, 'dropout_rate': 0.10813865831440783}. Best is trial 4 with value: 0.8355000019073486.
[I 2023-06-19 08:07:04,743] Trial 5 finished with value: 0.8383333086967468 and parameters: {'learning_rate': 0.028780037225536223, 'dropout_rate': 0.2295255139067653}. Best is trial 5 with value: 0.8383333086967468.
[I 2023-06-19 08:29:56,676] Trial 6 finished with value: 0.6629999876022339 and parameters: {'learning_rate': 0.0005534569928516209, 'dropout_rate': 0.23985322683450944}. Best is trial 5 with value: 0.8383333086967468.
[I 2023-06-19 08:52:36,691] Trial 7 finished with value: 0.43808332085609436 and parameters: {'learning_rate': 4.218766555468695e-05, 'dropout_rate': 0.47927284864315906}. Best is trial 5 with value: 0.8383333086967468.
[W 2023-06-19 08:57:39,287] Trial 8 failed with parameters: {'learning_rate': 0.08225272035842891, 'dropout_rate': 0.3715458718097801} because of the following error: KeyboardInterrupt().
Traceback (most recent call last):
  File "c:\Users\caiay\AppData\Local\Programs\Python\Python310\lib\site-packages\optuna\study\_optimize.py", line 200, in _run_trial
    value_or_values = func(trial)
  File "C:\Users\caiay\AppData\Local\Temp\ipykernel_8416\1648828260.py", line 35, in objective
    model.fine_tune(3, base_lr = learning_rate)
  File "c:\Users\caiay\AppData\Local\Programs\Python\Python310\lib\site-packages\fastai\callback\schedule.py", line 168, in fine_tune
    self.fit_one_cycle(epochs, slice(base_lr/lr_mult, base_lr), pct_start=pct_start, div=div, **kwargs)
  File "c:\Users\caiay\AppData\Local\Programs\Python\Python310\lib\site-packages\fastai\callback\schedule.py", line 119, in fit_one_cycle
    self.fit(n_epoch, cbs=ParamScheduler(scheds)+L(cbs), reset_opt=reset_opt, wd=wd, start_epoch=start_epoch)
  File "c:\Users\caiay\AppData\Local\Programs\Python\Python310\lib\site-packages\fastai\learner.py", line 264, in fit
    self._with_events(self._do_fit, 'fit', CancelFitException, self._end_cleanup)
  File "c:\Users\caiay\AppData\Local\Programs\Python\Python310\lib\site-packages\fastai\learner.py", line 199, in _with_events
    try: self(f'before_{event_type}');  f()
  File "c:\Users\caiay\AppData\Local\Programs\Python\Python310\lib\site-packages\fastai\learner.py", line 253, in _do_fit
    self._with_events(self._do_epoch, 'epoch', CancelEpochException)
  File "c:\Users\caiay\AppData\Local\Programs\Python\Python310\lib\site-packages\fastai\learner.py", line 199, in _with_events
    try: self(f'before_{event_type}');  f()
  File "c:\Users\caiay\AppData\Local\Programs\Python\Python310\lib\site-packages\fastai\learner.py", line 247, in _do_epoch
    self._do_epoch_train()
  File "c:\Users\caiay\AppData\Local\Programs\Python\Python310\lib\site-packages\fastai\learner.py", line 239, in _do_epoch_train
    self._with_events(self.all_batches, 'train', CancelTrainException)
  File "c:\Users\caiay\AppData\Local\Programs\Python\Python310\lib\site-packages\fastai\learner.py", line 199, in _with_events
    try: self(f'before_{event_type}');  f()
  File "c:\Users\caiay\AppData\Local\Programs\Python\Python310\lib\site-packages\fastai\learner.py", line 205, in all_batches
    for o in enumerate(self.dl): self.one_batch(*o)
  File "c:\Users\caiay\AppData\Local\Programs\Python\Python310\lib\site-packages\fastai\learner.py", line 235, in one_batch
    self._with_events(self._do_one_batch, 'batch', CancelBatchException)
  File "c:\Users\caiay\AppData\Local\Programs\Python\Python310\lib\site-packages\fastai\learner.py", line 199, in _with_events
    try: self(f'before_{event_type}');  f()
  File "c:\Users\caiay\AppData\Local\Programs\Python\Python310\lib\site-packages\fastai\learner.py", line 223, in _do_one_batch
    self._do_grad_opt()
  File "c:\Users\caiay\AppData\Local\Programs\Python\Python310\lib\site-packages\fastai\learner.py", line 211, in _do_grad_opt
    self._with_events(self._backward, 'backward', CancelBackwardException)
  File "c:\Users\caiay\AppData\Local\Programs\Python\Python310\lib\site-packages\fastai\learner.py", line 199, in _with_events
    try: self(f'before_{event_type}');  f()
  File "c:\Users\caiay\AppData\Local\Programs\Python\Python310\lib\site-packages\fastai\learner.py", line 207, in _backward
    def _backward(self): self.loss_grad.backward()
  File "c:\Users\caiay\AppData\Local\Programs\Python\Python310\lib\site-packages\torch\_tensor.py", line 478, in backward
    return handle_torch_function(
  File "c:\Users\caiay\AppData\Local\Programs\Python\Python310\lib\site-packages\torch\overrides.py", line 1551, in handle_torch_function
    result = torch_func_method(public_api, types, args, kwargs)
  File "c:\Users\caiay\AppData\Local\Programs\Python\Python310\lib\site-packages\fastai\torch_core.py", line 382, in __torch_function__
    res = super().__torch_function__(func, types, args, ifnone(kwargs, {}))
  File "c:\Users\caiay\AppData\Local\Programs\Python\Python310\lib\site-packages\torch\_tensor.py", line 1295, in __torch_function__
    ret = func(*args, **kwargs)
  File "c:\Users\caiay\AppData\Local\Programs\Python\Python310\lib\site-packages\torch\_tensor.py", line 487, in backward
    torch.autograd.backward(
  File "c:\Users\caiay\AppData\Local\Programs\Python\Python310\lib\site-packages\torch\autograd\__init__.py", line 200, in backward
    Variable._execution_engine.run_backward(  # Calls into the C++ engine to run the backward pass
KeyboardInterrupt
